cla_public/static-src/javascripts/modules/form-errors.js

```diff
@@ -11,12 +11,6 @@
       $('form')
         .on('submit', this.postToFormErrors)
         // Focus back on summary if field-error is focused and escape key is pressed
-        .on('keyup', function(e) {
-          var $target = $(e.target);
-          if(e.keyCode === 27 && ($target.is('.govuk-form-group--error') || $target.parent().is('.govuk-form-group--error'))) {
-            $target.closest('form').find('> .alert').focus();
-          }
-        })
```

Comment:
Do we want to replace the functionality or is it basically now redundant

Reply:
This is redundant.  Also it might clash with the escape key for quick exit.